### DISCUSSION TITLE
Website for Youtuber

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#35] [General]
Hey,

Long time no see.

I'm setting up a website for a Youtuber, and may need a little of your help, I would love to work on it myself, but I got exams coming up in like 11 days, so if you help, it can get done much faster.

Thanks,
Kendall Does Coding

**Comment by TechStudent10** [comment]
> I'm setting up a website for a Youtuber

Which YouTuber exactly?

> and may need a little of your help

Okay...

> but I got exams coming up in like 11 days, so if you help, it can get done much faster.

Good luck on your exams, but just because you have them doesn't mean you should forward the work to someone else. Besides, if they're in a week, that should be enough time to create, add and polish the website.

**Comment by KendallDoesCoding** [comment]
> I'm setting up a website for a Youtuber

> Which YouTuber exactly?

Small youtuber that ik irl.

> but I got exams coming up in like 11 days, so if you help, it can get
done much faster.

> Good luck on your exams, but just because you have them doesn't mean you
should forward the work to someone else. Besides, if they're in a week,
that should be enough time to create, add and polish the website.

Thanks! Yeah that's true. I was just thinking of a way to get it done
faster, but there's not much work on it anyway so I'll try to do it now, or
do it after my exams get over, there's no rush as of such, but just want to
get it done as soon as possible if you get what I mean.

> Besides, if they're in a week, that should be enough time to create, add
and polish the website.

Yeah, but I gotta do some serious studies lol

*Thanks,*
*Kendall Does Coding*

*Portfolio
<https://signal1domain.com/click?redirect=https%3A%2F%2Fkendalldoescoding.gq&dID=1647525882389&linkName=Portfolio>*
*Github
<https://signal1domain.com/click?redirect=https%3A%2F%2Fgithub.com%2FKendallDoesCoding&dID=1647525882389&linkName=Github>*
*Youtube*
<https://signal1domain.com/click?redirect=https%3A%2F%2Fkendalldoescoding.gq%2FcodingYT&dID=1647525882389&linkName=Youtube>

**Comment by KendallDoesCoding** [comment]
Hey,

I set up the website for the YouTuber on Wordpress: https://blitzrgaming.netlify.app/

**Reply by TechStudent10** [reply]
You should really get rid of the phone number and the email address if it they're personal/YouTube one. This would make them more prominent for scams, and maybe even hacking or SIM-swapping.

**Reply by KendallDoesCoding** [reply]
Yeah ik, the email is business one, the phone number he told me to add but yeah its a personal one.

**Reply by TechStudent10** [reply]
> the phone number he told me to add but yeah its a personal one.

Well then you should him that there is a risk of him getting hacked if he uses his phone number for MFA.

**Reply by KendallDoesCoding** [reply]
Ok, but the site isn't being released now coz he changed his mind.

**Reply by TechStudent10** [reply]
alright then
